docs/result.md

```diff
@@ -0,0 +1,257 @@
+# Pipeline Result Schema Reference
+
+The following document describes the various data dump outputs of the Reference Island data pipeline process, and the structure of the data within.
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [JSON and JSON-L dumps](#json-and-json-l-dumps)
+  - [Whitelisted External Ids Dump](#whitelisted-external-ids-dump)
+  - [Extracted Items Dump](#extracted-items-dump)
+  - [Potential Matches Dump](#potential-matches-dump)
+- [Lines](#lines)
+  - [`ItemLine`](#itemline)
+  - [`MatchLine`](#matchline)
+- [Blobs](#blobs)
+  - [`StatementBlob`](#statementblob)
+    - [Potential Values](#potential-values)
+  - [`ResourceBlob`](#resourceblob)
+  - [`ReferenceBlob`](#referenceblob)
+  - [`ReferenceMetaBlob`](#referencemetablob)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## JSON and JSON-L dumps
+
+### Whitelisted External Ids Dump
+
+This dump is the Result of [SS 1: External Resource Whitelister](pipeline.md#ss-1-external-resource-whitelister). It is a `json` file containing an array of Wikibase external id PID strings.
+
+[Scrape #3 Whitelisted Resources Example](https://raw.githubusercontent.com/wmde/reference-hunting-data/master/whitelisted_ext_idefs.json?token=ABOZJNMEXT7OJRQDOAVWDUC63JNUQ).
+
+### Extracted Items Dump
+
+This dump is the result of [Pipe 1: Item Extractor](pipeline.md#pipe-1-item-extractor). It is a `jsonl` file where each line follows the [`ItemLine`](#itemline) schema. 
+
+[Scrape #2 Item Dump Example](https://github.com/wmde/reference-hunting-data/raw/master/first_run/extracted_unreferenced_statements.jsonl.gz).
+
+### Potential Matches Dump
+
+This dump is the result of [Pipe 2: Scraper](pipeline.md#pipe-2-scraper), [Pipe 3: Value Matcher](pipeline.md#pipe-3-value-matcher) and [Pipe 4: Statistical Matcher](pipeline.md#pipe-4-statistical-matcher). It is a `jsonl` file where each line follows the [`MatchLine`](#matchline) schema.
+
+[Scrape #2 Potential Match Example](https://raw.githubusercontent.com/wmde/reference-hunting-data/master/second_run/references.jsonl?token=ABOZJNJRCIEWGXBVW5G4HAS63JN5U).
+
+## Lines
+
+### `ItemLine`
+
+A JSON object to represent a Wikibase item which contains unreferenced statements and [***whitelisted***](pipeline.md#ss-1-external-resource-whitelister) external source URLs. This information is extracted from a Wikidata data dump in [Pipe 1: Item Extractor](#pipe-1-item-extractor).
+
+```js
+{
+     "itemId": String // Wikibase QID of the item 
+     "statements": [ StatementBlob ] // An array of unreferenced statement claims
```

Comment:
Perhaps add `...` to the arrays to make clear than can be >1 in length

Reply:
:+1:

Reply:
On second thought, after adding it, it seems redundant, as we explicitly say it's an array straight after, and we consistently use `[]` notation to mark arrays.